src/main/java/qna/domain/Answer.java

```diff
@@ -29,45 +34,39 @@ public Answer(User writer, Question question, String contents) {
 
     public Answer(Long id, User writer, Question question, String contents) {
         super(id);
-
-        if(writer == null) {
+        if (writer == null) {
             throw new UnAuthorizedException();
         }
-
-        if(question == null) {
+        if (question == null) {
             throw new NotFoundException();
         }
-
         this.writer = writer;
         this.question = question;
         this.contents = contents;
     }
 
-    public Answer setDeleted(boolean deleted) {
-        this.deleted = deleted;
-        return this;
-    }
-
     public boolean isDeleted() {
         return deleted;
     }
 
-    public boolean isOwner(User writer) {
-        return this.writer.equals(writer);
-    }
-
     public User getWriter() {
         return writer;
     }
 
-    public String getContents() {
-        return contents;
-    }
-
     public void toQuestion(Question question) {
         this.question = question;
     }
 
+    DeleteHistory delete() {
+        DeleteHistory history = DeleteHistory.answerOf(getId(), writer);
+        deleted = true;
```

Comment:
```suggestion
        this.deleted = true;
```

사소하지만 여기도 Question에서 사용한 것 같이 통일시키면 좋을 것 같네요 😄

Reply:
넵 다음 단계에서 수정하겠습니다!